Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -901,6 +911,28 @@ - (void)setAccessibilityLabel:(NSString *)accessibilityLabel
 {
   _bridge_prologue_write;
   _setAccessibilityToViewAndProperty(_accessibilityLabel, accessibilityLabel, accessibilityLabel, accessibilityLabel);
+  if (AS_AT_LEAST_IOS11) {
+    _accessibilityAttributedLabel = [[NSAttributedString alloc] initWithString:accessibilityLabel];
+    BOOL shouldApply = ASDisplayNodeShouldApplyBridgedWriteToView(self);
+    if (shouldApply) {
+      [_view setValue: [[NSAttributedString alloc] initWithString:accessibilityLabel] forKey:@"accessibilityAttributedLabel"];
+    } else {
+      [ASDisplayNodeGetPendingState(self) setValue: [[NSAttributedString alloc] initWithString:accessibilityLabel] forKey:@"accessibilityAttributedLabel"];
+    }
+  }
+}
+
+- (NSAttributedString *)accessibilityAttributedLabel
+{
+  _bridge_prologue_read;
+  return _getAttributedAccessibilityFromViewOrProperty(_accessibilityAttributedLabel, @"accessibilityAttributedLabel");
+}
+
+- (void)setAccessibilityAttributedLabel:(NSAttributedString *)accessibilityAttributedLabel
+{
+  _bridge_prologue_write;
+  { _setAttributedAccessibilityToViewAndProperty(_accessibilityAttributedLabel, accessibilityAttributedLabel, @"accessibilityAttributedLabel", accessibilityAttributedLabel); }
+  { _setAttributedAccessibilityToViewAndProperty(_accessibilityLabel, accessibilityAttributedLabel.string, @"accessibilityLabel", accessibilityAttributedLabel.string); }
```

Comment:
Shouldn't we use `_setAccessibilityToViewAndProperty` for `_accessibilityLabel`?

Reply:
Hm, I wonder if one of the tests should have caught that. But you're definitely right!